Training/interface/DataLoader_main.h

```diff
@@ -0,0 +1,868 @@
+#include "TauMLTools/Core/interface/AnalysisMath.h"
+#include "TauMLTools/Analysis/interface/TauTuple.h"
+#include "TauMLTools/Training/interface/DataLoader_tools.h"
+
+#include "TROOT.h"
+#include "TLorentzVector.h"
+
+using namespace std;
+
+enum class CellObjectType {
```

Comment:
`CellObjectType` should be defined in setup.h. When generating setup, you need additionally generate helper class for features:
```c++
template<typename T>
struct FeaturesHelper;

template<>
FeaturesHelper<PfCand_electron_Features> {
    static constexpr CellObjectType object_type = CellObjectType::PfCand_electron;
    static constexpr size_t size = 2;
    using scaler_type = Scaling::PfCand_electron;
};

...
```

Reply:
I am a bit struggling with this inside .gInterpreter.ProcessLine() for some reasons

Reply:
and if you generate a temporary .h file and then `ProcessLine('#include "tmp.h"')`?

Reply:
With the temporary header it is working